src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings

```diff
@@ -806,6 +806,8 @@ prototype_file_noApplicableFileGroup = No Applicable Filegroup
 ############################################################################
 # Backup Service
 Backup_TaskName = Backup Database
+Backup_ServerCertificate = ServerCertificate;
```

Comment:
These strings are from SSMS, but I also don't think these need to be localized. I removed the strings, and instead changed to pass both the encryptor type and name.